mungegithub/features/aliases.go

```diff
@@ -72,33 +72,28 @@ func (a *Aliases) Name() string {
 
 // Initialize will initialize the feature.
 func (a *Aliases) Initialize(config *github.Config) error {
-	a.data = &aliasData{
-		AliasMap: map[string][]string{},
-	}
+	a.data = &aliasData{}
+	a.aliasReader = a
 
-	if len(a.aliasFile) == 0 {
-		return nil
+	if len(a.aliasFile) > 0 {
+		a.IsEnabled = true
```

Comment:
The alias feature is initialized if any enabled munger requests it and a munger that requests the alias feature could be used without an alias file being provided. This shouldn't be an error, an empty AliasMap is used and aliases just aren't expanded. IsEnabled is totally pointless though, I got rid of it.